### ISSUE TITLE
Adding the C-terminus OC1 and OC2 to the default list of hydrogen bond acceptors

### ISSUE THREAD

**Issue by xiki-tempula** [MDAnalysis/mdanalysis#1342]
In Gromcas, if you use the amber99sb-ildn force field. The two oxygens in the C terminus of the protein are named as OC1 and OC2, which is named as O originally. These two oxygens are part of the carboxylic acid and are hydrogen bond acceptors.
Currently the atom name OC1 and OC2 are not in the default list of hydrogen bond acceptors. I wonder if it is possible to add these two atoms into the default list of hydrogen bond acceptors? As far as I'm concern, the name OC1 and OC2 only represent the C-terminal oxygen.

### Expected behaviour
If the C-terminus of a protein make hydrogen bond with another thing, the hydrogen bond is detected.
### Actual behaviour
It is not.
### Code to reproduce the behaviour

``` python
from MDAnalysis import Universe
from MDAnalysis.analysis.hbonds.hbond_analysis import HydrogenBondAnalysis
from six import StringIO

gro = '''test
3
    1ALA    OC1    1   0.000   0.000   0.000
    2ALA      N    2   0.300   0.000   0.000
    2ALA     H1    3   0.200   0.000   0.000
7.29748 7.66094 9.82962'''
u = Universe(StringIO(gro), format='gro')

h = HydrogenBondAnalysis(u)
h.run()
print(h.timeseries)
# [[]]

h = HydrogenBondAnalysis(u, acceptors = ['OC1',])
h.run()
print(h.timeseries)
# [[[3, 1, 2, 0, 'ALA2:H1', 'ALA1:OC1', 2.0, 180.0]]]

```

### Currently version of MDAnalysis:
py36 mda0.16.1

**Comment by orbeckst** [comment]
submit a PR, please, that's easy to add
